merfi/backends/rpm_sign.py

```diff
@@ -39,6 +40,16 @@ def clear_sign(self, path, command):
         """
         logger.info('signing: %s' % path)
         out, err, code = util.run_output(command)
+        # Sometimes rpm-sign will fail with this error. I've opened a ticket
+        # with RH secalert. For now, sleep and retry as a workaround.
+        known_failure = "ERROR: unhandled exception occurred: ('')."
+        tries = 1
+        while known_failure in err and tries < 10:
+            logger.warning('hit known rpm-sign failure.')
+            tries += 1
+            logger.warning('sleeping, running try #%d in 30 seconds.' % tries)
```

Comment:
it is safe to use a comma instead of `%` to prevent breaking from badly formatted log lines:

    logger.warning('sleeping, running try #%d in 30 seconds.', tries)

Reply:
This is a difference with merfi's logger, see https://github.com/alfredodeza/merfi/pull/64

Reply:
who wrote this garbage logger thing